freebsd_sysctl/__init__.py

```diff
@@ -201,9 +201,9 @@ def query_fmt(oid: typing.List[int]) -> bytes:
         if len(buf) < 4:
             raise Exception("response buffer too small")
         result = buf[:buf_length]
-        kind, = struct.unpack("<I", result[:4])
+        kind, = struct.unpack("I", result[:4])
```

Comment:
You could apply the same change to value conversion in CtlType. In fact it would solve a newly discovered issue when unpacking unpadded values.

https://github.com/gronke/py-freebsd_sysctl/blob/9cdc892d4792b6a5189bd1476660684a9b90acb8/freebsd_sysctl/types.py#L50